### PR TITLE
Auth page initial design

### LINKED ISSUE #21: Create auth page <Client>

### LINKED ISSUE #22: Create the client auth page v1

### PR DESCRIPTION
**Pull request by Ayobami6**

### CODE REVIEW

File: server/controllers/offerController.ts
Comment (betiniakarandut) [inline server/controllers/offerController.ts:1]: This is great bro. Nice work!

File: server/controllers/offerController.ts [hunk 1]
```diff
@@ -1,23 +1,11 @@
-import { Request, Response } from 'express';
-// import { Document } from 'mongoose';
```
Comment (betiniakarandut) [inline server/controllers/offerController.ts:2]: Impressive work! Sorry bro, but should not we remove this line (line 2) just to keep the code clean and simple as it is?

File: server/test/routes/offers.test.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    let app;
+    beforeAll(async () => {
+        await connectDB();
+    });
+    beforeEach(async () => {
+        app = express();
+        await app.use(express.json());
+        await app.use('/offers', offerRoutes);
+    });
+    afterAll(async () => {
```
Comment (betiniakarandut) [inline server/test/routes/offers.test.ts:29]: Interesting one here. I love the combination.

File: server/test/dbTestConnection.ts
Comment (betiniakarandut) [inline server/test/dbTestConnection.ts:1]: Wow! Nice and brilliant work

**Review by betiniakarandut — APPROVED** [review]
Excellent work here!

**Comment by Ayobami6** [comment]
Thanks 👍

File: server/controllers/offerController.ts (hunk 1 above)
Comment (Ayobami6) [reply server/controllers/offerController.ts:2]: Yeah, we definitely should

**Review by jojothomas1515 — APPROVED** [review]
Doing wonders

**Comment by Utilitycoder** [comment]
Great work guys!

**Pull request merged**